### PR TITLE
Added option to generate different genre of jokes

### PR DESCRIPTION
**Pull request by kartikeyjangir**
Added option to generate different genre of jokes and also added nodemon package

### CODE REVIEW

**Comment by kartikeyjangir** [comment]
Added option to generate different genre of jokes and added .env file to hide api keys

File: app.js [hunk 1]
```diff
@@ -3,20 +3,53 @@ var request = require("request");
...
-            data = JSON.parse(body);
+            data = {
+                body:JSON.parse(body).body[0],
+                urlName:"dadJokes"
+                };
+            res.render("getJokes",{data: data});
+        }
+    })
+});
+app.get("/chuck", function(req, res) {
```
Comment (ExpressHermes) [inline app.js:32]: Making a separate endpoint to go to get different jokes is not ideal. If we add 5-6 genres, we will end up with 5-6 endpoints all doing the same thing(calling an API and showing results). Make all the changes in the same endpoint and use functions and conditions to show the result.

One way is when a genre is selected by the user(user clicked Chuck Norris joke button on the home page, we can use the button to send input back to the nodejs app to understand which genre user wants and then show them the jokes based on that.

File: package.json [hunk 2]
```diff
@@ -4,14 +4,16 @@
   "description": "Gives random jokes",
   "main": "app.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "start": "nodemon app.js"
```
Comment (ExpressHermes) [inline package.json:7]: Remove `nodemon`. This is okay for development not for prod.

File: views/getJokes.ejs [hunk 3]
```diff
@@ -1,24 +1,24 @@
 <% include partials/header %>
-<% var jokeType = data.type %>
 
 <div class="ui container">
     <div class="ui raised centered yellow card">
         <div class="content">
-            <% if(jokeType ==="single") { %>
+             <% if(data.urlName==="dadJokes") { %>
```
Comment (ExpressHermes) [inline views/getJokes.ejs:6]: this if-else statement assumes that there are only two types of jokes that can be returned(Dad Jokes and Chuck Norris). Instead, we will have more types of jokes. So use only if statement for each type of joke without using else.

Pseudocode:
```
if(isDadJoke) {
    // show the joke
}

if (isChuckNorrisJoke) {
    ...
}

if (anothTypeOfJoke) {
    ...
}
```

File: views/partials/jokeTypes.ejs [hunk 4]
```diff
@@ -0,0 +1,5 @@
+
+<div>
+    <button style="margin-bottom:20px; background-color: #4ECDC4; border: none; border-radius: 10px; font-size: 22px;"> <a href="/" style="color: white;">Get Dad Jokes</a></button>
```
Comment (ExpressHermes) [inline views/partials/jokeTypes.ejs:3]: Use css file instead of inline css

**Review by ExpressHermes — CHANGES_REQUESTED** [review]
Please make the requested changes!

**Comment by ExpressHermes** [comment]
Also, remove `package-lock.json` file.

**Pull request closed without merging**